app/controllers/api/v1/games_controller.rb

```diff
@@ -1,16 +1,26 @@
 module Api::V1
   class GamesController < ApplicationController
 
+    before_action :conflict_error, if: -> { lobby.game.present? },
```

Comment:
semantically, I'm not sure 409 is a perfect fit here. Seems like 405 or just 400 might be more accurate.